app/models/pregnancy.rb

```diff
@@ -33,6 +33,15 @@ class Pregnancy
   # Validations
   validates :created_by,
             presence: true
+  before_update :check_other_validations?
```

Comment:
Ahh you are so close! We should try and do this with validations rather than hooks if we can. Here's a suggested potential way that use the `check_other_validations?` method you wrote:

* Exclusions: http://guides.rubyonrails.org/active_record_validations.html#exclusion
Something like `exclusion: in: [ true ], unless: check_other_validations?` (In plain english: 'Value cannot be true, unless check other validations is also true.`